src/univariate.jl

```diff
@@ -66,8 +66,7 @@ function kde_range(boundary::Tuple{Real,Real}, npoints::Int)
     lo, hi = boundary
     lo < hi || error("boundary (a,b) must have a < b")
 
-    step = (hi - lo) / (npoints-1)
-    lo:step:hi
+    linspace(lo, hi, npoints)
```

Comment:
`linspace` is deprecated in 0.7, so this should be `Compat.range(lo, stop=hi, length=npoints)`

Reply:
Good catch.